contracts/ReversibleICO.sol

```diff
@@ -617,9 +610,7 @@ contract ReversibleICO is IERC777Recipient {
         view
```

Comment:
One questions to the `getCancelModeStates()` function. What if we would disapprove a user and the reprove him, because of something would this function break?